Sources/MarshaledObject.swift

```diff
@@ -15,8 +15,8 @@ import Foundation
 
 
 public protocol MarshaledObject {
-    subscript(key: KeyType) -> Any? { get }
     func anyForKey(_ key: KeyType) throws -> Any
+    func optionally(key: KeyType) -> Any?
```

Comment:
this name feels weird... attack of the adverb again? `synchronously`? 😛 

How about something like what Elixir and Elm do: `get`?